python/private/py_info.bzl

```diff
@@ -67,6 +67,11 @@ the venv to create the path under.
 
 A runfiles-root relative path that `venv_path` will symlink to. If `None`,
 it means to not create a symlink.
+""",
+        "src": """
```

Comment:
No comment on renaming this field?

Reply:
hmmm. I renamed this to `package`, I liked the suggestion.